tekton/resources/checks/check-pr-has-kind-label.yaml

```diff
@@ -0,0 +1,218 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: check-labels-trigger
+rules:
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["tekton.dev"]
+  resources: ["eventlisteners", "triggerbindings", "triggertemplates", "tasks", "taskruns"]
```

Comment:
The tasks and taskruns resources here are not necessary

Reply:
Thanks, changed